models/user.js

```diff
@@ -1,10 +1,16 @@
 const mongoose = require('mongoose');
+const Schema = mongoose.Schema;
 
 const userSchema = new mongoose.Schema(
     {
       name: String,
       email: String,
       googleId: String,
+      exercises: [{
+        type: Schema.Types.ObjectId,
+        ref: 'Log'
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/Nlopez4/fitness_tracking_app/pullRequest?prid=7841248)
```suggestion
        ref: "Log"
```